nodes/quantum/classical-register/classical-register.js

```diff
@@ -89,7 +89,7 @@ module.exports = function(RED) {
           done(err);
         } else {
           quantumCircuitConfig[node.name] = register;
-          send(output);
+          //send(output);
```

Comment:
This send(output) here causes error, since there is no such reference here and classical register logically send no message. That is why comment it out.

Reply:
This has been fixed in master. Merge master into your branch to verify that all tests are working before merging this PR.